apps/src/templates/maker/DiscountCodeSchoolChoice.jsx

```diff
@@ -45,16 +46,23 @@ export default class DiscountCodeSchoolChoice extends Component {
       confirming: true
     });
 
-    // TODO: eventually this will be an API call. To start with, just fake it
-    // by at least making it async
-    setTimeout(() => {
-      this.setState({
-        confirming: false,
-        confirmed: true,
-      });
-      const fullDiscount = false;
-      this.props.onSchoolConfirmed(fullDiscount);
-    }, 1000);
+    $.ajax({
+     url: "/maker/schoolchoice",
+     type: "post",
+     dataType: "json",
+     data: {
+       nces: this.state.schoolId
+     }
+   }).done(data => {
+     this.props.onSchoolConfirmed(data.full_discount);
+     this.setState({
+       confirming: false,
+       confirmed: true,
+     });
+   }).fail((jqXHR, textStatus) => {
+     // TODO: should probably introduce some error UI
+     console.error(textStatus);
```

Comment:
These TODOs tend to stick around forever.  Can we at least `setState({confirming: false, confirmed: false})` in this case so the user could try again?